src/components/charts/bubble-chart/bubble-chart-styles.scss

```diff
@@ -1,7 +1,29 @@
+@import '~styles/settings.scss';
+
 .circle {
   cursor: pointer;
 }
 
-.tooltip span {
-  text-align: left !important;
+:global {
+  .bubbleChartTooltip {
+    font-family: $font-family-1;
+    padding: 15px;
+    font-size: $font-size-xs;
+    letter-spacing: -0.1px;
+    box-shadow: 0 5px 15px 0 rgba(17, 55, 80, 0.15);
+    max-width: 300px;
+
```

Comment:
Not sure why but background opacity is not 1. It will be better this way

Reply:
Ok, I will change it